Dockerfile

```diff
@@ -0,0 +1,12 @@
+## Dockerfile
+
+FROM node:alpine
+RUN mkdir -p /opt/app
+WORKDIR /opt/app
+RUN adduser -S app
+COPY . .
+RUN yarn install
+RUN chown -R app /opt/app
```

Comment:
to use docker cache It's better to split the installation of dependencies from the rest of the code.

```suggestion
# Install only dependencies
COPY --chown=node:node package.json yarn.lock /opt/app/
# force yarn global cache to live in node_modules or will duplicate files
RUN yarn --global-folder ./node_modules/ --cache-folder ./node_modules/

# Install the rest of the code
COPY --chown=node:node . /opt/app
```